tests/integration/docker-compose.yml

```diff
@@ -49,6 +52,17 @@ services:
       MINIO_ROOT_PASSWORD: "password"
       MINIO_CONSOLE_ADDRESS: ":9001"
 
+  google-storage:
+    image: fsouza/fake-gcs-server
+    ports:
+      - 4443:4443
+    entrypoint: sh
+    command: >
+      -c 'mkdir -p /storage/bucket1 && 
+          mkdir -p /storage/bucket2 && 
+          mkdir -p /storage/bucket3 && 
```

Comment:
buckets should be created/deleted by the tests themselves

Reply:
oh, let me try. this `fake` guys almost killed me because of stupid configuration so I really don't want to touch anything but let me try of course.

Reply:
done.